gcpkms/keysource.go

```diff
@@ -220,12 +222,18 @@ func (key *MasterKey) newKMSClient() (*kms.KeyManagementClient, error) {
 		opts = append(opts, option.WithCredentialsJSON(key.credentialJSON))
 	default:
 		credentials, err := getGoogleCredentials()
-		if err != nil {
-			return nil, err
-		}
 		if credentials != nil {
 			opts = append(opts, option.WithCredentialsJSON(credentials))
 		}
+
+		at_credentials, at_err := getGoogleAccessToken()
+		if at_credentials != nil {
+			opts = append(opts, option.WithTokenSource(at_credentials))
```

Comment:
Will it cause any issues if _both_ the default credentials and access token credentials are added to `opts`?